assets/js/analyzer.js

```diff
@@ -1,5 +1,6 @@
 var analyzers = {}, analyzerData = {};
 var currentAnalyzerRequest;
+var THRESHOLD_REQUEST_LENGTH_FOR_ANALYZE = 2020;
```

Comment:
@sushain97 ,I initially planned to keep only a single `AJAX` function. Depending upon the endpoint I supply as the parameter, I would handle the `success` accordingly! And almost all of the rest of the parameters of the `AJAX` method would stay the same (which they are even now!). Do you think this is a good idea? A lot of code is repetitive in the current state.